docs/schemas/shared/IfcSharedBldgElements/Types/IfcWallTypeEnum.md

```diff
@@ -24,7 +24,7 @@ A movable wall that is either movable, such as folding wall or a sliding wall, o
 A wall-like barrier to protect human or vehicle from falling, or to prevent the spread of fires. Often designed at the edge of balconies, terraces or roofs, or along edges of bridges.
 
 ### PARTITIONING
-A wall designed to partition spaces that often has a light-weight, sandwich-like construction (e.g. using gypsum board). Partitioning walls are normally non load bearing.
+A wall designed to partition spaces that can be a light-weight, sandwich-like construction (e.g. using gypsum board) or massive walls made out of blocks or elements (e.g. using gipsym, light-weight concrete of calcium silicate). Partitioning walls are normally non load bearing.
```

Comment:
```suggestion
A wall designed to partition spaces that can be a light-weight, sandwich-like construction (e.g. using gypsum board) or massive walls made out of blocks or elements (e.g. using gypsum, light-weight concrete of calcium silicate). Partitioning walls are normally non load bearing.
```

Reply:
I also have some doubts about the word "massive". I know where it's coming from, I'm also Dutch, but maybe "solid" or "homogeneous" are clearer?

Other than that, I think this is a good clarification 👍

Reply:
Yes, I think a native English speaker should rewrite it correctly.

Reply:
What about this? I'm also not a native English speaker...
```suggestion
A wall designed to partition spaces that can be a light-weight, layered construction (e.g. using gypsum board) or solid walls made out of blocks or elements (e.g. using gypsum, light-weight concrete or calcium silicate). Partitioning walls are non-load-bearing.
```

Reply:
@jbrouwerdigibase looks good to me